milestone-delivery/deliveries/SaintsUndone-M2.md

```diff
@@ -0,0 +1,18 @@
+# Milestone Delivery :mailbox:
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:** [Saints Undone Grant Application PR](https://github.com/vechain/grant-program/pull/185). 
+* **Milestone Number:** 2
+
+The team has dedicated considerable effort towards expediting the development of the solution, striving to fulfill each promised deliverable diligently. Despite encountering challenges during our company registration journey, we have navigated through them, and our organization is now poised on the brink of finalizing its registration as a Private Limited Company in India.
+
+* **Updates on Deliverables:**
+
+| Number | Deliverable | Specification / Link | Note | Status |
+| ------------- | ------------- | ------------- |------------- | ------------- |
+| 0a. | Company | Registration of Company | The necessary formalities have been completed, and we are now awaiting the GST Number from the Ministry of Corporate Affairs, Government of India | :heavy_check_mark: |
+| 0b.  | Documentation |[Documentation](https://docs.google.com/document/d/1dHKDjbXXfLjqSURBZpU5kovWm2QaPtMe7TsnGVPDu1Q/edit?usp=sharing)| Combining NFC-NFT | :heavy_check_mark: |
+| 1.  | NFC Tag Integration |[Video Demo](https://youtu.be/N_0PEbjX36g)| Demo Video | :heavy_check_mark: |
+| 2.  | NFT Minting Enhancement | [Saints_Undone_WebApp](https://github.com/Pranav9931/saints-undone-webapp) | Github Hash: 9cdce6c265cf9bcb7e5b7bbcbc7e430168c35025 | :heavy_check_mark: |
```

Comment:
I can not access the GitHub repo you have posted.

Reply:
The repo is private, let me know your GitHub ID, I'll add you to the repo.

Or do you suggest to make it public?

Reply:
We encourage projects to make their public repo, the decision is yours. In this case, we only need to access to evaluate the delivery.

Reply:
Better provide your GitHub ID then.
We are a Private Limited Company and It would be better if we do not make our repo public as there are multiple contents right now and will be in future that is not meant to be public.
Thanks!

Reply:
https://github.com/libotony

Reply:
Done, I've given you access to the repos.
Thanks!